client/app/bundles/shared/components/Textarea.scss

```diff
@@ -0,0 +1,52 @@
+@import "base.scss";
+
+$focus-background-color-darken: rgba(0, 0, 0, 0.4);
+$focus-background-color-lighten: lighten($container-color, 3%);
+$light-font-weight: 300;
+
+.label {
+  color: $container-color;
+  font-family: $font-family;
+  font-size: $small-font;
+  padding: 12px 4px;
+  letter-spacing: 0.0944444em;
```

Comment:
Should this be in px?

Reply:
I think you left a similar comment before, and this was nshki's response: https://github.com/ifmeorg/ifme/pull/792/files/bddb09bfe3dba6058c8b07fe5fc3a3c441ec3f4b#diff-8c6a9053fe95ea482db2fa6012fd737e

I ended up assuming that ems were okay. But if not, let me know!

Reply:
Oh true! Okie dokes, em is fine then :P Should we round up the value though @nshki?

Reply:
Can't hurt to round to 0.095!

Reply:
Haha sure thing! Adjusted the line in Input.scss as well for good measure :)